HackYourFuture-CPH/fp-class19
# Get favourite endpoint

# Description

Created an endpoint to get all favorites products for specified user.

Fixes #26

# How to test?

Pull the Branch, 
Go to your db and insert a row into table favourites,
Run: npm run dev ,
Check endpoint like [http://localhost:3000/api/user/user_id/favorites
go to: [http://localhost:3000/api/documentation] to check swagger documentation for the endpoint,

![image](https://user-images.githubusercontent.com/81110602/152567127-3b2df82f-8093-4df1-add7-8d6ef950a050.png)

# Checklist
